src/components/BackButton/BackButton.stories.js

```diff
@@ -0,0 +1,8 @@
+import BackButton from './BackButton';
+
+export default {
+  title: 'BackButton',
+  component: BackButton,
+};
+
+export const Default = () => <BackButton />;
```

Comment:
If you would have different versions of BackButtons, you could add them here as further `const` like `SmallBackButton` etc.